README.md

```diff
@@ -1,92 +1,224 @@
+For more details browse our [daal4py documentation](https://intelpython.github.io/daal4py/).
+
+
+## oneDAL Samples
+
+<img align="right" style="display:inline;" height=300 width=550 src="docs/readme-charts/intel%20oneDAL%20Spark%20samples%20vs%20Apache%20Spark%20MLlib.png"></a>
+
+Samples is a non official part of a project, that contains examples of how oneDAL can be used in different applications.
+
+**oneDAL Spark Samples is a good example.**
+oneDAL provides scala / java interfaces that match Apache Spark MlLib API and use oneDAL solvers under the hood. This implementation allows you to get a 3-18X increase in performance compared to default Apache Spark MLlib.
+
+  ## Examples
+
+Except C++ and Python API oneDAL also provide API for C++ SYCL and Java languages. Check out tabs below for more examples.
+- [C++](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/cpp)
+- [C++ SYCL\*](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/cpp_sycl)
+- [Java](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/java)
+- [Python](https://github.com/IntelPython/daal4py/tree/master/examples)
 
 Data Examples for different computation modes:
 
-- [Batch](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/batch)
-- [Distributed](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/distributed)
-- [Online](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/online)
+- [Batch](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/data/batch)
+- [Distributed](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/data/distributed)
+- [Online](https://github.com/oneapi-src/oneDAL/tree/master/examples/daal/data/online)
 
 ## Documentation
-
 - [Get Started](http://oneapi-src.github.io/oneDAL/getstarted.html)
-- [System Requirements](https://software.intel.com/en-us/articles/system-requirements-for-oneapi-data-analytics-library#)
+- [System Requirements](https://software.intel.com/content/www/us/en/develop/articles/system-requirements-for-oneapi-data-analytics-library.html)
 - [oneDAL documentation](http://oneapi-src.github.io/oneDAL/)
 - [Specifications](https://spec.oneapi.com/versions/latest/elements/oneDAL/source/index.html)
-- [Release Notes](https://software.intel.com/en-us/articles/oneapi-dal-release-notes)
+- [Release Notes](https://software.intel.com/content/www/us/en/develop/articles/oneapi-dal-release-notes.html)
 - [Known Issues](https://oneapi-src.github.io/oneDAL/notes/known_issues.html)
 
-## API
+## oneDAL and Intel&reg; DAAL
+
+Intel&reg; oneAPI Data Analytics Library is an extenstion of Intel&reg; Data Analytics Acceleration Library (Intel&reg; DAAL).
 
-Intel(R) DAAL [provides](https://software.intel.com/en-us/articles/daal-api-reference) downloadable API References for C++, Python, and Java.
+This repository contains branches corresponding to both oneAPI and classical versions of the library. We encourage you to use oneDAL located under the `master` branch.
```

Comment:
I think we also should add mention on internal split even within single branch - like oneDAL have old APIs and newAPIs

Reply:
This can be handled in separate PRs